SECURITY.md

```diff
@@ -1,4 +1,4 @@
-<!--security-start-->
+<!-- SECURITY_INC_START -->
```

Comment:
This is a bad change. No change is needed here. This change causes breakage. See the what that this file is included by `/docs/content/contribution-guidelines/security/security-inc.md`.